src/Sylius/Behat/Context/Setup/UserContext.php

```diff
@@ -130,6 +130,7 @@ private function createUser(
         $user->setUsername($email);
         $user->setEmail($email);
         $user->setPlainPassword($password);
+        $user->enable();
```

Comment:
I would say that usually we would like to have enabled user account as default. If we need to make it disable we can add a separate step for that `And its account is disabled`.